server/core/router.js

```diff
@@ -12,16 +12,22 @@ const registerWallet = require('../controllers/registerWallet');
 const cryptoInfo = require('../controllers/cryptoInfo');
 const walletBalance = require('../controllers/walletBalance');
 const walletCoinBalance = require('../controllers/walletCoinBalance');
-
 const generateApiKey = require('../controllers/generateApiKey');
+const coinFullForm = require('../controllers/coinFullForm');
 
 //  Internal APIs
 router.post('/wallet/register', registerWallet);
 
+//  Placeholder API
+router.get('/', (req, res) => {
+    res.status(200).json({ msg: 'Hi!' });
+});
+
 //  Kucoin APIs (TODO: Make it accept any coded exchange)
 router.get('/convert_currency', cryptoInfo);
 router.get('/:userId/wallet/balance', walletBalance);
 router.get('/:userId/wallet/:coin/balance', walletCoinBalance);
 //router.post('/create_api_key', generateApiKey);
+router.post('/coin_full_form', coinFullForm);
```

Comment:
Não deveria ser POST isso aqui. POST a gente só usa quando cadastra algo lembra? Tem que ser um GET. Ai você passa o nome da moeda por query ou path parameter (me chama se não entender isso)

Reply:
Também acho que o path coin_full_form não é uma rota muito clara, que tal /coin/name? Aí você cria um domínio /coin  se a gente for criar alguma outra análise em cima de uma coin colocaria nesse domínio, tipo /coin/timeline

Reply:
Noice